Runtime/AvatarManifest.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+using ReadyPlayerMe.Core;
+using UnityEngine;
+
+namespace ReadyPlayerMe.AvatarLoader
+{
+    public class AvatarManifest
+    {
+        private static readonly string RELATIVE_PATH = "/AvatarManifest.json";
+        private Dictionary<string, DateTime> avatarRecords = new Dictionary<string, DateTime>();
+        public Dictionary<string, DateTime> AvatarRecords => avatarRecords;
+
+        public DateTime GetAvatarLastLoadDate(string guid)
+        {
+            return avatarRecords.TryGetValue(guid, out DateTime date) ? date : DateTime.UtcNow;
+        }
+
+        public void AddAvatar(string guid)
+        {
+            if (!avatarRecords.ContainsKey(guid))
```

Comment:
Small nitpick, and I'm not aware of your agreed conventions yet so feel free to ignore me if you've agreed to do things a certain way. However, for me, code is much easier to read with the minimal amount of nesting possible, so generally I'd normally write this as:

```suggestion
            if (avatarRecords.ContainsKey(guid))
               return;
               
            avatarRecords.Add(guid, DateTime.UtcNow);
```

Reply:
Yeah I get it, it is kinda subjective as to whether it is more or less readable. 
I don't really have any strong feelings about either approach, but generally the code our SDK doesn't do it this way. We can of course change but if we do we should change it everywhere to ensure consistency within the code.

Reply:
Agreed. Only noted it in the hope of prompting a conversation about convention, but 100% aligned that consistency is the most important.